config/config.go

```diff
@@ -973,6 +978,8 @@ type InhibitRule struct {
 	SourceMatchRE MatchRegexps `yaml:"source_match_re,omitempty" json:"source_match_re,omitempty"`
 	// SourceMatchers defines a set of label matchers that have to be fulfilled for source alerts.
 	SourceMatchers Matchers `yaml:"source_matchers,omitempty" json:"source_matchers,omitempty"`
+	// Sources defines a set of source matchers and equal labels.
+	Sources []Source `yaml:"source,omitempty" json:"source,omitempty"`
```

Comment:
The yaml/json naming is missing an `s` suffix:
```suggestion
	// Sources defines a set of source matchers and equal labels.
	Sources []Source `yaml:"sources,omitempty" json:"sources,omitempty"`
```

I'm also not fully happy with this naming, but still think about alternatives:
- `AllSourceMatchers`
- `MultiSourceMatchers`
- etc.

Maybe other maintainers have a better idea how to name this thing.